TYPO3.Flow/Classes/TYPO3/Flow/Object/Proxy/ProxyInterface.php

```diff
@@ -17,12 +17,4 @@
  */
 interface ProxyInterface
 {
-    /**
-     * Wake up method.
-     *
-     * Proxies need to have one as at least session handling relies on it.
-     *
-     * @return void
-     */
-    public function __wakeup();
```

Comment:
Well "Proxies need to have one as at least session handling relies on it" is a hint, but of course that might be outdated. Does Flow call `__wakeup` explicitly (without checking it is callable) in any place?